pkg/util/clusterdata/worker_profile.go

```diff
@@ -102,10 +102,10 @@ func (ce machineClientEnricher) Enrich(
 	oc.Lock.Lock()
 	defer oc.Lock.Unlock()
 
-	oc.Properties.WorkerProfiles = workerProfiles
```

Comment:
after further thinking (thanks pto), I believe this will break our admin api. Since the doc will not have `workerProfiles` set, when the conversion is made, workerProfiles is nil and so the output will not have workerProfiles since we first check for workerProfiles and then check for workerProfileStatus. Any automation in place which relies on the workerProfiles via admin API would need to be changed.

I think an alternative approach, would be to leave the doc the same (always have it be enriched via workerProfiles) and for new API versions we can do the conversion.

Reply:
Updated workerProfiles references as suggessted by Ben.